src/main/kotlin/it/unibo/lss/fcla/reservation/domain/entities/reservation/Aim.kt

```diff
@@ -0,0 +1,17 @@
+package it.unibo.lss.fcla.reservation.domain.entities.reservation
+
+import it.unibo.lss.fcla.reservation.domain.entities.exceptions.WorkoutReservationAimCannotBeEmpty
+
+/**
+ * Aim value object.
+ *
+ * It is the unique identifier of a workout [value]
+ */
+data class Aim(val value: String) {
+
+    init {
+        if (value.isEmpty()) {
+            throw WorkoutReservationAimCannotBeEmpty()

```

Comment:
Is this exception right? How does the Aim know that it refers to a WorkoutReservation?